reconcile/prometheus_rules_tester.py

```diff
@@ -297,6 +327,11 @@ def check_prometheus_tests(tests, rules, thread_pool_size, settings):
 
         try:
             for cluster, namespaces in rules[rule_files[0]].items():
+                if len(target_clusters) > 0 and cluster not in target_clusters:
+                    logging.info(
```

Comment:
```suggestion
                    logging.debug(
```
I think we tend to be more minimal on the INFO lines...

Reply:
changed to debug.